plugins/modules/device_configs_backup_workflow_manager.py

```diff
@@ -339,6 +397,74 @@ def get_device_ids_list(self):
         self.log("Backup of {0} devices out of {1} devices is possible".format(valid_device_count, original_valid_device_count), "INFO")
         return device_ids
 
+    def get_site_devices(self):
```

Comment:
get_devices_by_site_and_params 

better reflects the functionality of fetching device IDs based on both site and other parameters as per your API documentation

Reply:
Done